### PR TITLE
Enable mountd_port parameter for Red Hat.

### PR DESCRIPTION
**Pull request by rnelson0**

### CODE REVIEW

File: manifests/server/redhat.pp [hunk 1]
```diff
@@ -13,10 +18,15 @@
   }
 
   if ($mountd_port != undef){
-    fail('Setting mountd port currently not supported on RedHat')
+    shellvar { 'rpc-mount-options':
+      ensure   => present,
+      target   => '/etc/sysconfig/nfs',
+      variable => 'MOUNTD_PORT',
+      value    => "${mountd_port}",
```
Comment (dwerder) [inline manifests/server/redhat.pp]: manifests/server/redhat.pp - WARNING: string containing only a variable on line 25

Thats a reason why travis CI shows an error.

**Comment by rnelson0** [comment]
Fixed the issue on line 25.

Aside: .gitignore doesn't include Gemfile.lock. Do you want me to include that in this PR in the same or separate commit or fix it in a separate PR?

**Comment by dwerder** [comment]
You can add it to this PR. Thanks

**Comment by rnelson0** [comment]
@dwerder Whew, I think I abused TravisCI enough to discover the fix to the future parser issues and such. This should be ready for review/merge now!

**Comment by rnelson0** [comment]
@dwerder Thanks! What's the timeframe for releasing a new version on the forge? Just need to know when I can modify the Puppetfile back to the forge ref.

**Comment by dwerder** [comment]
I will trey to release it next week. I am on vacation at the moment ;)

**Comment by rnelson0** [comment]
Bad Daniel! :) Enjoy your vacation, I'll point to github for now.


Rob Nelson
rnelson0@gmail.com

**Comment by rnelson0** [comment]
Bad Daniel! Enjoy your vacation, I'll point to github for now :)


Rob Nelson
rnelson0@gmail.com

**Pull request merged**